tests/unit/auth/unauthenticated.ts

```diff
@@ -0,0 +1,83 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Not sure if there is a better way but this file feels a bit out of place 🤔 
It is in the auth service tests but it tests the configurations service. 
Would `tests/unit/client/client.test.ts` be a better place to put these tests?